packages/webapp/components/FooterNavBar.tsx

```diff
@@ -122,8 +126,19 @@ export default function FooterNavBar({
   };
 
   return (
-    <div className="fixed !bottom-0 left-0 z-2 w-full">
-      <ScrollToTopButton />
+    <div
+      className={classNames(
+        'fixed !bottom-0 left-0 z-2 w-full',
+        post && 'bg-blur-bg backdrop-blur-20',
+      )}
+    >
+      {post ? (
+        <div className="mb-2 w-full px-2 tablet:hidden">
+          <NewComment post={post} />
+        </div>
+      ) : (
+        <ScrollToTopButton />
+      )}
```

Comment:
On mobile we show the new comment button just above the footer navbar. To avoid problems we previously had with scroll to top buttons not rendering in the correct spot on mobile Safari, I added the button directly to navbar, similar to the scroll to top button.